### PR TITLE
Minor code improvements/rewrites

### PR DESCRIPTION
**Pull request by yojance**
Some light refactoring of code

### CODE REVIEW

**Review by yojance — COMMENTED** [review]
is_phone() and prepare_line_items() will break if you use this code. I overlooked the fact that is return false, within the if statement.

**Comment by yojance** [comment]
First time doing a pull request. Please see my notes above, 

is_phone() and prepare_line_items() will break if you use this code. I overlooked the fact that it return false, within the if statement instead of returning true

File: includes/class-wc-validation.php [hunk 1]
```diff
@@ -32,11 +32,7 @@ public static function is_email( $email ) {
 	 * @return  bool
 	 */
 	public static function is_phone( $phone ) {
-		if ( 0 < strlen( trim( preg_replace( '/[\s\#0-9_\-\+\/\(\)]/', '', $phone ) ) ) ) {
-			return false;
-		}
-
-		return true;
+		return 0 < strlen( trim( preg_replace( '/[\s\#0-9_\-\+\/\(\)]/', '', $phone ) ) );
```
Comment (claudiosanches) [inline includes/class-wc-validation.php]: Changed in entire logic here, now returns false when should return true.

File: includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php [hunk 2]
```diff
@@ -312,11 +312,7 @@ protected function prepare_line_items( $order ) {
 		}
 
 		// Check for mismatched totals.
-		if ( $this->number_format( $calculated_total + $order->get_total_tax() + $this->round( $order->get_shipping_total(), $order ) - $this->round( $order->get_total_discount(), $order ), $order ) != $this->number_format( $order->get_total(), $order ) ) {
-			return false;
-		}
-
-		return true;
+		return $this->number_format( $calculated_total + $order->get_total_tax() + $this->round( $order->get_shipping_total(), $order ) - $this->round( $order->get_total_discount(), $order ), $order ) != $this->number_format( $order->get_total(), $order );
```
Comment (claudiosanches) [inline includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php]: Changed the logic here too.

**Comment by claudiosanches** [comment]
> is_phone() and prepare_line_items() will break if you use this code. I overlooked the fact that it return false, within the if statement instead of returning true

So just fix it.
If not is impossible to merge.

**Pull request closed without merging**